packages/block-editor/src/store/reducer.js

```diff
@@ -922,161 +922,177 @@ export function isCaretWithinFormattedText( state = false, action ) {
 	return state;
 }
 
-const BLOCK_SELECTION_EMPTY_OBJECT = {};
-const BLOCK_SELECTION_INITIAL_STATE = {
-	start: BLOCK_SELECTION_EMPTY_OBJECT,
-	end: BLOCK_SELECTION_EMPTY_OBJECT,
-	isMultiSelecting: false,
-	isEnabled: true,
-	initialPosition: null,
-};
-
 /**
- * Reducer returning the block selection's state.
+ * General reducer returning the block selection's start and end.
```

Comment:
Perhaps just to me, but _general reducer_ isn't enough to convey that this is only an internal helper for two other reducers. Can we make it more explicit? Maybe:

> Internal helper reducer for `selectionStart` and `selectionEnd`. Can hold a block selection, represented by an object with property `clientId`.